### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by mvandenburgh**
Adds
	- Note about where to find `manage.py` scripts documentation to README
	- Instructions for running celery beat to README (which is now required for things to work correctly locally)
	- celery beat as a service in `docker-compose.override.yml`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,14 +48,17 @@ but allows developers to run Python code on their native system.
...
    1. `source ./dev/export-env.sh`
    2. `./manage.py runserver`
 3. Run in a separate terminal:
    1. `source ./dev/export-env.sh`
-   2. `celery --app dandiapi.celery worker --loglevel INFO --without-heartbeat`
-4. When finished, run `docker-compose stop`
+   2. `celery --app dandiapi.celery worker --loglevel INFO --without-heartbeat -Q celery,calculate_sha256 -B`
+4. Run in a seperate terminal:
+   1. `source ./dev/export-env.sh`
+   2. `celery --app dandiapi.celery beat --loglevel INFO`
```
Comment (dchiquito) [inline README.md:60]: These should not be necessary when using the `-B` flag to run the worker.

File: docker-compose.override.yml [hunk 2]
```diff
@@ -39,3 +39,22 @@ services:
...
+      "--loglevel", "INFO"
+    ]
+    tty: false
+    env_file: ./dev/.env.docker-compose
+    volumes:
+      - .:/opt/django-project
+    depends_on:
+      - postgres
+      - rabbitmq
+      - minio
```
Comment (dchiquito) [inline docker-compose.override.yml:60]: This shouldn't be necessary when using the `-B` flag on `celery_worker`.

**Review by dchiquito — CHANGES_REQUESTED** [review]
I think the separate beat workers shouldn't be necessary for development. Is the `-B` flag not triggering beats for you @mvandenburgh ?

**Comment by mvandenburgh** [comment]
> I think the separate beat workers shouldn't be necessary for development. Is the `-B` flag not triggering beats for you @mvandenburgh ?

Oh wow it does. I've been running celery beat as a separate process until now :exploding_head:

**Review by dchiquito — APPROVED** [review]
Looks good, I thought I had updated all the celery invocations.

**Pull request merged**